datashader/geo.py

```diff
@@ -0,0 +1,320 @@
+from __future__ import division
```

Comment:
Please add a module docstring explaining that this module is optional and that it provides utilities for working with geographic data.

Reply:
done..